src/App.js

```diff
@@ -1,51 +1,79 @@
 import React from 'react';
+import { isBoolean } from 'util';
+
 import TodoList from './components/TodoComponents/TodoList'
 import TodoForm from './components/TodoComponents/TodoForm'
-import Todo from './components/TodoComponents/Todo'
+
+const taskObjects = [
+  {
+      task: 'clean',
+      id: Date.now(),
+      completed: false,
+  },
+  {
+      task: 'mop',
+      id: Date.now(),
+      completed: false,
+  },
+  {
+      task: 'nap',
+      id: Date.now(),
+      completed: false,
+  },
+]
 
 class App extends React.Component {
+  // you will need a place to store your state in this component.
+  // design `App` to be the parent component of your application.
+  // this component is going to take care of state, and any change handlers you need to work with your state
   constructor() {
     super();
     this.state = {
-
+      taskObjects: taskObjects,
+      task: '',
+      id: '',
+      completed: isBoolean,
     };
   }
+  
+    addTask = e => {
+      e.preventDefault();
+      const newTask = {
+        task: this.state.task,
+        id: this.state.id,
+        completed: this.state.completed,
+      };
+      this.setState({
+        taskObjects: [...this.state.taskObjects, newTask],
+        task: '',
+        id: '',
+        completed: isBoolean,
+      });
+    };
 
-  addTask = e => {
-    e.preventDefault();
-    console.log(e.target);
-    const newTask = {
-      task: this.state.task,
-      id: this.state.id,
-      completed: this.state.completed,
+    listTask = e => {
+      this.setState({
+        [e.target.task]: e.target.value
```

Comment:
Tomorrow I can show you what is up with this - this is a little tricky here and once I explain it you will see what is going on.